plugins/notifications-backend/dev/index.ts

```diff
@@ -96,7 +100,7 @@ const notificationsDebug = createBackendPlugin({
           });
           res.status(200).send({ status: 'ok' });
         });
-        router.use(errorHandler());
+        router.use(middleware.error());
```

Comment:
Could we completely get rid of this middleware?

Reply:
If I understood the debate earlier correctly, the default "root" handler doesn't get the logging context right, so it doesn't look obvious that the log line comes from this particular plugin. But I'm happy if that's not the case so we can delete these!